lib/rules/forbid-component-props.js

```diff
@@ -31,20 +31,40 @@ module.exports = {
         forbid: {
           type: 'array',
           items: {
-            type: 'string'
+            oneOf: [{
+              type: 'string'
+            }, {
+              type: 'object',
+              properties: {
+                propName: {
+                  type: 'string'
+                },
+                allowedFor: {
```

Comment:
let's make sure this list is unique

Reply:
done; didn't know json schema has an option to require uniqueness